src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -94,7 +94,8 @@ const FlatTableRow = React.forwardRef(
     if (onClick || expandable) {
       interactiveRowProps = {
         isRowInteractive: !firstColumnExpandable,
-        tabIndex: firstColumnExpandable || isSubRow ? undefined : 0,
+        tabIndex:
+          (firstColumnExpandable || isSubRow) && !onClick ? undefined : 0,
```

Comment:
This is going to break the `firstColumnExpandable` prop if the row has also been given an `onClick` prop - the `onClick` should only apply to the first column in that situation. 

Changing this to
```suggestion
          firstColumnExpandable ? undefined : 0,
```
should be what we want here

Reply:
Yep my bad, missed that it would set on the main expandable row as well